### PR TITLE
Add license and other requirements for CRAN

### PR DESCRIPTION
**Pull request by greebie**
Some minor changes to set up for cran (e.g. add documentation for default).
image

### CODE REVIEW

**Comment by greebie** [comment]
This version has been submitted to CRAN as archivr-0.0.1.   Once this is merged, I will push a release to complete the project.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#' Copyright <2019> <Qualitative Data Repository, Syracuse University>
```
Comment (adam3smith) [inline LICENSE]: Any reason you're using the #' `roxygen` comments? Since this is in a separate file, the license can just be plain text, no?

Comment (greebie) [reply LICENSE]: No reason to use comments I don't think.

Comment (adam3smith) [reply LICENSE]: Will it mess with your CRAN submission if I take these out? Otherwise this is good to go.

Comment (greebie) [reply LICENSE]: No I don't think so. Once it's accepted (or not) I can repair it accordingly.

**Pull request merged**